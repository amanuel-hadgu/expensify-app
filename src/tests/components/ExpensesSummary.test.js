import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should correctly render ExpensesSummary with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={324} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});

test('should correctly render ExpensesSummary with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={123} expensesTotal={32412324563} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});