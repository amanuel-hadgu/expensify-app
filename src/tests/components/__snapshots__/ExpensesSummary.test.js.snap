// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    123
     
    expenses
     totalling 
    $324,123,245.63
  </h1>
</div>
`;

exports[`should correctly render ExpensesSummary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $3.24
  </h1>
</div>
`;
